apps/sms/views/inbox/js/inbox.js

```diff
@@ -803,6 +823,20 @@ var InboxView = {
     }
   },
 
+  /**
+   * Update the DOM element for a thread.
```

Comment:
__nit:__ you've added ```conversationDraft``` property, but use ```thread``` in new jsdoc all over the place - is it intentionally or we should use ```conversation``` everywhere?